src/ingestion/sources/base.py

```diff
@@ -337,7 +337,8 @@ def _col_sql(self, item, select_body):
 
         if col_sql is not None:
             select_body[table_name]['sql'].append(col_sql)
-        if col_type == 'date' or col_type == 'timestamp_tz':
+        # TODO: Better handling of date / time types is needed 
+        if col_type == 'date' or "timestamp" in col_type:
```

Comment:
Nasty bug that was causing no timestamps to return. This should cover most of the cases but may require better handling

Reply:
Not sure why this is nasty, therefore can't tell you what to do here, but I'm certain this breaks other things. Please elaborate

Reply:
Basically the timestamp column was not being recognized because we were only checking that it was equivalent to `date` or `timestamp_nz`. The problem is that these are not the only two time stamp types that can be encountered. I ran into this bug and called it nasty because spent a while debugging it.

Ultimately, most timestamp column types either contain the `date` string or the `timestamp` string, so we are checking for the substring rather than the direct equivalence.